x/ibc/applications/transfer/keeper/msg_server.go

```diff
@@ -27,17 +27,21 @@ func (k Keeper) Transfer(goCtx context.Context, msg *types.MsgTransfer) (*types.
 
 	k.Logger(ctx).Info("IBC fungible token transfer", "token", msg.Token.Denom, "amount", msg.Token.Amount.String(), "sender", msg.Sender, "receiver", msg.Receiver)
 
-	ctx.EventManager().EmitEvents(sdk.Events{
-		sdk.NewEvent(
-			types.EventTypeTransfer,
-			sdk.NewAttribute(sdk.AttributeKeySender, msg.Sender),
-			sdk.NewAttribute(types.AttributeKeyReceiver, msg.Receiver),
-		),
+	if err := ctx.EventManager().EmitTypedEvent(
+		&types.EventTransfer{
+			Sender:   msg.Sender,
+			Receiver: msg.Receiver,
+		},
+	); err != nil {
+		return nil, err
+	}
+
+	ctx.EventManager().EmitEvent(
 		sdk.NewEvent(
 			sdk.EventTypeMessage,
```

Comment:
I think this event can be removed. Pretty sure the message type and module name are emitted in baseapp